EdgeOfEmpireBot/EdgeOfEmpireBot/Service/MessageService.cs

```diff
@@ -0,0 +1,113 @@
+﻿using Discord.WebSocket;
+using EdgeOfEmpireBot.IService;
+
+namespace EdgeOfEmpireBot.Service
+{
+    public class MessageService : IMessageService
+    {
+        private bool messageIsSafe = false;
+
+        /// <inheritdoc/>
+        public async Task SendMessage(string messageResponse, ISocketMessageChannel messageResponseChannel)
+        {
+            var message = string.Empty;
+  
+            // Discord API prevents messages greater than 2000 characters from sending.
+            if (messageResponse.Length > 2000)
+            {
+                message = "```ERROR 2319 - Message is greater than the maximum 2000 character limit.```";
+                Console.WriteLine(message);
+            }
+            else
+            {
+                message = messageResponse;
+            }
+
+            await messageResponseChannel!.SendMessageAsync(message);
+
+        }
+
+        /// <inheritdoc/>
+        public async Task ProcessMessage(SocketMessage message)
```

Comment:
Its better off doing a guard clause here.
If(!message.Content.StartsWith(".")){ ignore message }

then you wont need to do an else statement.

Reply:
Good call, do we want to rule out messages with other deliminaters though? for secret/debug references. 

ie '>debug' or '>log' instead of '.debug', '.log'. I know '.log' is spammed sometimes on PBTA.

Reply:
I reckon at the moment we just focus on getting the commands working for the one deliminater first then we can focus on creating new ones for secrets/debug first

Reply:
Fair enough, I've updated to be a guard cluase